operators/bevel.py

```diff
@@ -137,19 +137,24 @@ def main(self, context):
         l1, l2 = connected
         self.connected = connected
 
-        intersections = _get_intersections(
-            *_get_offset_elements(l1, radius),
-            *_get_offset_elements(l2, radius),
+        # If more than 1 intersection point, then sort them so we prioritise 
+        # the closest ones to the selected point.
```

Comment:
I'm not sure about that. The closest one doesn't have to be the correct one. It will probably often be the case but not when creating a big bevel on a symmetrical setup like described in #267.

Reply:
I feel like that isn't possible to create that case with straight lines and arcs.

I could relatively easily create a solution for any arc-line combination for bevel where you can come up with the path along which the centerpoint moves (which is an arc I think) and use the radius with the function [`distance_along_segment`](https://github.com/hlorus/CAD_Sketcher/blob/main/class_defines.py#L1531) to get the centerpoint.

That allows to create a bevel along the whole length of both entities too, instead of half of them in some scenario.

However, it ain't really a general solution either, and a general solution sounds like it would be done on the library side, either slvs or blender (bpy) itself, but would require native blender objects.

So unless we want to maintain some very complex code that already exists in other libraries, i think this solution or the other one I suggested should be considered.